### DISCUSSION TITLE
Повысить лимит на количество изображений

### DISCUSSION THREAD

**Discussion by Saaneek0** [MrZillaGold/VK2Discord#428] [Q&A] [answered]
Очень понравилось ваше приложение VK2Discord!
Но такой вопрос. Можно ли как-то расширить кол-во прикрепляемых фотографий (я так понял сейчас прикрепляется до четырех фото из вк. Можно ли это как-то расширить? До 6 например, или до 10 сразу как в том-же вк?

**Comment by Silence02ru** [comment, marked as answer]
Тут уже нужно писать самому дискорд, с их стороны ограничение в 4 изображения

**Reply by Saaneek0** [reply]
> Тут уже нужно писать самому дискорд, с их стороны ограничение в 4 изображения

А, даже так, понял. Спасибо)
